share/smack/top.py

```diff
@@ -843,7 +843,7 @@ def transform_out(args, old):
     return out
 
 
-def verification_result(verifier_output):
+def verification_result(verifier_output, verifier):
```

Comment:
Is this new argument not used? It looks like `args.verifier` is used in the body.

Reply:
Good catch.

Reply:
Looks good to me.